core/templates/i18n/i18n.service.ts

```diff
@@ -81,8 +84,38 @@ export class I18nService {
   initialize(): void {
     this.i18nLanguageCodeService.onI18nLanguageCodeChange.subscribe(
       (code) => {
+        const cookieSetDateMsecs = this.cookieService.get(
```

Comment:
We had trouble using cookies for storing i18n information. Look at https://github.com/oppia/oppia/pull/14266. Can localStorage be used instead?